core/src/main/kotlin/com/tschuchort/compiletesting/KotlinCompilation.kt

```diff
@@ -520,7 +520,7 @@ class KotlinCompilation : AbstractKotlinCompilation<K2JVMCompilerArguments>() {
 
 			val javacCommand = jdkBinFile.absolutePath + File.separatorChar + "javac"
 
-			val isJavac9OrLater = isJavac9OrLater(getJavacVersionString(javacCommand))
+			val isJavac9OrLater = isJavac9OrLater()
```

Comment:
I don't think that this is correct. The system property would always contain the version name of the system JDK or at least the JDK that the program is run with, right? If the `KotlinCompilation` is given a `jdkHome` that points somewhere else, then that JDK's version would be different from the `java.specification.version`.